.github/workflows/build-artifacts.yml

```diff
@@ -28,12 +28,17 @@ jobs:
     - name: Build library
       run: bundle exec rake compile
 
+    - name: Package files
+      run: |
+        mkdir dist
+        cp -r include dist/
+        cp build/libprism.a dist/
+        tar -czf prism-${{ matrix.os }}.tar.gz -C dist .
+
     - name: Upload to release
       if: github.event_name == 'release'
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       uses: softprops/action-gh-release@v2
       with:
-        files: |
-          build/libprism.so
-          build/libprism.dylib
+        files: prism-${{ matrix.os }}.tar.gz
```

Comment:
I am not sure if this will be enough, since for mac and linux we still need to differentiate between `arm64` and `amd64` platforms, etc.

Besides, IMO `prism-macos-latest.tar.gz` is a weird name for the package.

Reply:
Based on [this chart](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories), it looks like to run amd64 macOS we need to add `macos-13` to the `os` matrix too.